resources/META-INF/scala-plugin-common.xml

```diff
@@ -920,6 +920,10 @@
                          displayName="Concealed &quot;apply&quot; call" groupPath="Scala" groupName="Syntactic Clarification"
                          shortName="ConcealedApplyCall" level="WARNING"
                          enabledByDefault="false" language="Scala"/>
+        <localInspection implementationClass="org.jetbrains.plugins.scala.codeInspection.syntacticClarification.ConvertNullInitializerToUnderscore"
+                         displayName="Null initializer can be replaced by _" groupPath="Scala" groupName="Syntactic Clarification"
+                         shortName="ConvertNullInitializerToUnderscore" level="WARNING"
+                         enabledByDefault="true" language="Scala"/>
```

Comment:
I'm not sure it's a good idea to enable it by default. For people who are coming from C-like languages, `null` is more obvious than `_`. It is clearer that if you call methods on it, it will throw a `NullPointerException`. In addition, a lot of code will be highlighted yellow after that.

@niktrop @pavelfatin what do you think?